cdbot/cogs/admin.py

```diff
@@ -0,0 +1,48 @@
+import re
+
+from cdbot.constants import NICKNAME_PATTERNS, PLACEHOLDER_NICKNAME
+from discord.ext.commands import Bot, Cog
+
+
+def checkName(nick):
+    for i in NICKNAME_PATTERNS:
+        if re.match(i, nick):
+            return True
+    return False
+
+
+class Admin(Cog):
+    """
+    Admin functionality
+    """
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @Cog.listener()  # triggered on new/removed nickname
+    async def on_member_update(memberBefore, memberAfter):
+        newNickname = memberAfter.display_name
```

Comment:
Surprised our linters didn't pick up on this, but could we replace `camelCase` with `snake_case` thanks

Reply:
This applies to functions as well as variables

Reply:
Oh sorry, can do.